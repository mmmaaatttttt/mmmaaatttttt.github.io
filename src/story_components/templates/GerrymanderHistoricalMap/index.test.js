import React from "react";
import { render } from "@testing-library/react";
import { PureHistoricalMap } from ".";

describe("smoke and snapshot tests", () => {
  it("renders successfully", () => {
    render(<PureHistoricalMap />);
  });

  it("renders first demonstration successfully", () => {
    const { asFragment } = render(<PureHistoricalMap />);
    expect(asFragment()).toMatchSnapshot();
  });
});
