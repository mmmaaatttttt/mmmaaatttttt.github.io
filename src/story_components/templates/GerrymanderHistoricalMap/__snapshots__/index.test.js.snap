// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke and snapshot tests renders first demonstration successfully 1`] = `
<div>
  <SliderProvider
    column={true}
    fullWidthAt="small"
    initialData={
      Array [
        Object {
          "color": "#555555",
          "initialValue": 2016,
          "max": 2016,
          "min": 1996,
          "step": 2,
          "title": [Function],
        },
        Object {
          "color": "#555555",
          "initialValue": 2,
          "max": 10,
          "min": 2,
          "step": 1,
          "title": [Function],
        },
      ]
    }
    render={[Function]}
    width="100%"
  />
</div>
`;
